python/plugins/phidata/composio_phidata/toolset.py

```diff
@@ -28,39 +28,60 @@ def _wrap_tool(
         self,
         schema: t.Dict,
         entity_id: t.Optional[str] = None,
-    ) -> Function:
+    ) -> Toolkit:
         """
-        Wrap composio tool as Phidata `Function` object.
+        Wrap composio tool as Phidata `Toolkit` object.
         """
         name = schema["name"]
         description = schema["description"]
         parameters = schema["parameters"]
-
+        
+        # Create a new Toolkit instance
+        toolkit = Toolkit(name=name)
```

Comment:
Consider adding error handling for cases where the schema is invalid or missing required fields. This would make the code more robust and provide better error messages.